ingester/user_state.go

```diff
@@ -28,10 +29,11 @@ type userState struct {
 	ingestedSamples *ewmaRate
 }
 
-func newUserStates(rateUpdatePeriod time.Duration) *userStates {
+func newUserStates(rateUpdatePeriod time.Duration, maxSeriesPerUser int) *userStates {
```

Comment:
Hmm, `rateUpdatePeriod` is not a limit (it's only used to be able to show rates in the UI), so `UserLimitsConfig` doesn't fit so well. Maybe just `UserStatesConfig` since the only commonality is to configure user states? But then at two arguments, I think it makes more sense to keep it arguments?